main.py

```diff
@@ -73,7 +73,32 @@
 
     await bot.process_commands(message)
 
+#start CuckTim Section
+#replaces "Cuck" with Rotating Light Emoji, meant to piss off Tim
+@bot.event
+async def on_message(message):
+    if not message.author.bot and "cuck" in message.content.lower():
+        CuckString = message.content.replace('cuck', ':rotating_light:')
+        newString = re.compile("cuck", re.IGNORECASE)
+        await bot.send_message(message.channel, newString.sub('cuck', cuckString))
```

Comment:
What you wrote will replace the word "cuck" with "cuck" which is pretty useless.

The string literal on this line should be ':rotating_light:'